### ISSUE TITLE
Can't find modules fs and path

### ISSUE THREAD

**Issue by xzilja** [rickbutton/yoga-wasm#6]
I am using this with webpack 5 which no longer includes polyfills for fs and path, hence am getting errors  about this during build time. are they necessary within browser environment? Perhaps they should be polyfiled at a package lvl?

**Comment by rickbutton** [comment]
Yeah, this is likely because of the way that I am packaging this package. wasm modules must be loaded, and without native support for wasm-es modules the way to load them is via node's fs, which doesn't exist in the browser obviously.

I need to research the right way to package an environment independent wasm module + loader.
